daemon/api/endpoints/partial/flow.py

```diff
@@ -51,16 +51,18 @@ async def _create(flow: 'FlowModel', ports: Optional[PortMappings] = None):
 )
 async def _update(
     kind: UpdateOperation,
-    dump_path: str,
     pod_name: str,
-    shards: int = None,
+    uses_with: str,
```

Comment:
[`uses_with` should be a `Dict[str, str]`](https://fastapi.tiangolo.com/tutorial/body-nested-models/?h=dict#bodies-of-arbitrary-dicts).

Reply:
In theory is a Dict of str and anything, would that be possible to implement?